net/stubby/files/patch-CMakeLists.diff

```diff
@@ -0,0 +1,36 @@
+--- CMakeLists.txt.orig	2021-01-17 18:41:14.000000000 -0500
++++ CMakeLists.txt	2021-01-17 18:42:23.000000000 -0500
+@@ -51,7 +51,7 @@
+   GROUP_READ GROUP_EXECUTE
+   WORLD_READ WORLD_EXECUTE
+   )
+-set(STUBBYCONFDIR "${CMAKE_INSTALL_FULL_SYSCONFDIR}/stubby")
++set(STUBBYCONFDIR "%%STUBBYCONFDIR%%")
+ 
+ find_package(Libyaml REQUIRED)
+ 
+@@ -115,15 +115,15 @@
+ 
+ # Ensure the file gets CRLF line endings on Windows.
+ file(GENERATE OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/stubby.yml INPUT ${CMAKE_CURRENT_SOURCE_DIR}/stubby.yml.example)
+-# Copy to destination iff no destination file exists.
+-install(CODE "\
+-  set(targetdir ${STUBBYCONFDIR})\n\
+-  if (\$ENV{DESTDIR})\n\
+-    set(targetdir \$ENV{DESTDIR}/\${targetdir})\n\
+-  endif ()\n\
+-  if (NOT EXISTS \${targetdir}/stubby.yml)\n\
+-    file(COPY ${CMAKE_CURRENT_BINARY_DIR}/stubby.yml DESTINATION \${targetdir})\n\
+-  endif ()")
++## Copy to destination iff no destination file exists.
++#install(CODE "\
++#  set(targetdir ${STUBBYCONFDIR})\n\
++#  if (\$ENV{DESTDIR})\n\
++#    set(targetdir \$ENV{DESTDIR}/\${targetdir})\n\
++#  endif ()\n\
++#  if (NOT EXISTS \${targetdir}/stubby.yml)\n\
++#    file(COPY ${CMAKE_CURRENT_BINARY_DIR}/stubby.yml DESTINATION \${targetdir})\n\
++#  endif ()")
```

Comment:
Why add a patch that comments out code? Why not add a patch that removes the code? It's smaller and easier to read.